_projects/en/designers.md

```diff
@@ -0,0 +1,111 @@
+
+The site [designers.italia.it](https://designers.italia.it/) contains:
+
+* References to the current version of the [Design Guidelines](https://designers.italia.it/guide/)
+* A series of [kits for the design of services of the Public Administration](https://designers.italia.it/kit/)
+* The [blog](https://designers.italia.it/blog/) with use case of the kits
+
+The source code of the site itself is hosted in a [GitHub repository](https://github.com/italia/designers.italia.it) open to any kind of contribution.
+
+## UI Kit
+
+The UI Kit is a set of visual components already ready to build the interface of a site or an app, following the guidelines for the digital services of the Public Administration. The kit is built with Sketch and defined in an open and collaborative way on GitHub together with the designer community. The UI Kit is the reference point for UI components and patterns intended to be implemented using the Web Development Kits, shown below.
+
+* [Introduction to the UI Kit](https://designers.italia.it/kit/ui-kit/)
+* [UI Kit components and patterns](https://invis.io/RJFGS2UC3HS)
+* [GitHub Repository](https://github.com/italia/design-ui-kit)
+
+Many Public Administration projects are already using the UI Kit to define their own interfaces, including models for [sites of the municipalities](https://github.com/italia/design-comuni-prototipi) and [sites of schools](https://github.com/italia/design-scuole-prototipi).
+
+For questions and discussions about the UI Kit, on [Developers Italia Slack](https://slack.developers.italia.it/), you can refer to the channel [#design-ui](https://developersitalia.slack.com/messages/C9N62GX8E/)
+
+## Web Development Kit
+
+The [Web Development Kits](https://designers.italia.it/kit/web-development-kit/) are a set of tools for the front end development of websites and web applications. They are HTML, CSS and Javascript libraries that allow you to create sites that comply with the Design Guidelines and code the provisions of the UI Kit. As of today, the most stable, updated and mature web development kit is Bootstrap Italia, based on the open-source library Bootstrap 4.
+
+### Bootstrap Italia
+
+The Bootstrap 4 based kit is the reference point for the development of public administration projects. The main purpose is to make available to the community of developers, in line with the [Design Guidelines](https://docs.italia.it/italia/designers-italia/design-linee-guida-docs/), syntax, tools, plugins and conventions of an instrument already known and used by many, such as [Bootstrap 4] (https://getbootstrap.com/).
```

Comment:
Public Administration con le maiuscole.
Il link a bootstrap è rotto, forse c'è uno spazio di troppo in mezzo.